### ISSUE TITLE
Upgrade swift-argument-parser to 1.0.0

### ISSUE THREAD

**Issue by AlexGustafsson** [maticzav/swift-graphql#47]
ArgumentParser has been updated to 1.0.0. Updating this packet will allow users to use the latest version of ArgumentParser in their SwiftPM builds.

**Comment by AlexGustafsson** [comment]
```
error: Dependencies could not be resolved because root depends on 'swift-graphql' 2.2.1..<2.3.0.
'swift-graphql' >= 2.0.0-beta.6 cannot be used because 'swift-graphql' >= 2.0.0-beta.6 depends on 'swift-argument-parser' 0.3.0..<1.0.0 and root depends on 'swift-argument-parser' 1.0.0..<1.1.0.
```

**Comment by maticzav** [comment]
This has been fixed #52 .
